packages/libs/error-stack/src/fmt/mod.rs

```diff
@@ -130,13 +137,15 @@
 //! [`SpanTrace`]: tracing_error::SpanTrace
 //! [`error_stack::fmt::builtin_debug_hook_fallback`]: crate::fmt::builtin_debug_hook_fallback
 //! [`atomic`]: std::sync::atomic
+//! [`Error::provide`]: std::error::Error::provide
+//! [`Frame.request_value()`]: Frame::request_value
+//! [`Frame.request_ref()`]: Frame::request_ref
+//! [`.attach()`]: Report::attach
```

Comment:
They should be in scope

```suggestion
```

Reply:
fixed in [d388a1b](https://github.com/hashintel/hash/pull/974/commits/d388a1b9f48f3acad5d61e3a7afd6dabf371897e)